packages/devtools-core/tests/components/DevTools/RenderLogPanel/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import "@babel/polyfill";
```

Comment:
I don't think we need this anymore since we have `useBuiltIns: "usage"` in the `.babelrc`. I Could be wrong though